monkey/infection_monkey/telemetry/attack/usage_telem.py

```diff
@@ -5,9 +5,9 @@ class UsageTelem(AttackTelem):
 
     def __init__(self, technique, status, usage):
         """
-        T1035 telemetry.
+        :param technique: Id of technique
         :param status: ScanStatus of technique
-        :param usage: Usage string
+        :param usage: Enum name of UsageEnum
```

Comment:
Why not just get the usage enum itself, instead of the enum.name.
It will be much cleaner to call, and might even be more convenient for processing.
call will be `T1035Telem(status, UsageEnum.SMB).send()` instead of `T1035Telem(status, UsageEnum.SMB.name).send()`

Reply:
![image](https://user-images.githubusercontent.com/36815064/62271093-66603b00-b440-11e9-9284-16c068a7cc97.png)